sky/shell/platform/android/org/domokit/sky/shell/FlutterSemanticsToAndroidAccessibilityBridge.java

```diff
@@ -228,6 +228,8 @@ private void sendAccessibilityEvent(int virtualViewId, int eventType) {
             event.setPackageName(mOwner.getContext().getPackageName());
             event.setSource(mOwner, virtualViewId);
             mOwner.getParent().requestSendAccessibilityEvent(mOwner, event);
+            // TODO: You probably want to call event.setText here even though
+            // it's redundant. Other fields are needed for text editing.
```

Comment:
Once we send accessibility events for text-related stuff I'll refactor this method to handle setting text on them.
It wasn't clear to me what it would mean to call setText on, say, AccessibilityEvent.TYPE_WINDOW_CONTENT_CHANGED.